app/views/admin/index.html.erb

```diff
@@ -1 +1,18 @@
-<%= render partial: "admin_nav" %>
\ No newline at end of file
+<%= render partial: "admin_nav" %>
+<br>
+<br>
+<%# Refactor this later to potentially add table or article? Maybe font size?  %>
```

Comment:
we kind of got away with this by utilizing float, but maybe still worth coming back to later